### DISCUSSION TITLE
Can not get same solution provided on sample code

### DISCUSSION THREAD

**Discussion by zhugelijuan123** [google/or-tools#3080] [Routing (and legacy CP) questions]
https://developers.google.com/optimization/routing/cvrp?hl=en 
I copy the entire code and run on my own desktop. I got the following solution which is different to solution on the website:
![image](https://user-images.githubusercontent.com/19217683/149449004-dc1f7db7-7ce2-45d7-a791-3fe08584c8fc.png)
Objective: 6208
Route for vehicle 0:
 0 Load(0) ->  4 Load(4) ->  3 Load(6) ->  1 Load(7) ->  7 Load(15) ->  0 Load(15)
Distance of the route: 1552m
Load of the route: 15

Route for vehicle 1:
 0 Load(0) ->  14 Load(4) ->  16 Load(12) ->  10 Load(14) ->  9 Load(15) ->  0 Load(15)
Distance of the route: 1552m
Load of the route: 15

Route for vehicle 2:
 0 Load(0) ->  12 Load(2) ->  11 Load(3) ->  15 Load(11) ->  13 Load(15) ->  0 Load(15)
Distance of the route: 1552m
Load of the route: 15

Route for vehicle 3:
 0 Load(0) ->  8 Load(8) ->  2 Load(9) ->  6 Load(13) ->  5 Load(15) ->  0 Load(15)
Distance of the route: 1552m
Load of the route: 15

Total distance of all routes: 6208m
Total load of all routes: 60

**Comment by Mizux** [comment]
May I know what is the problem ? sorry I can't find the diff between both results...

**Reply by zhugelijuan123** [reply]
that's both my result. It's different from the result on ortools website belong to capacity constraint, I provide link at the top of my question.

**Reply by zhugelijuan123** [reply]
also, there is another discussion I post named "question about customized constraints in vrp problem". I would appreciate a lot if you can give me some advice today since I'm trying to solve this this whole week and got really confused.

**Comment by zhugelijuan123** [comment]
that's both my result. It's different from the result on ortools website belong to capacity constraint, I provide link at the top of my question.

**Comment by Mizux** [comment]
Many thx !
should be fixed now: https://developers.google.com/optimization/routing/cvrp#solution
